docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -5,9 +5,20 @@
 <titleabbrev>Synthetic monitoring</titleabbrev>
 ++++
 
-The {uptime-app} in {kib} lets you monitor user journeys and full page loads using our browser monitor,
-and also monitor the availability and response times of applications and services using our lightweight HTTP/S,
-TCP, and ICMP monitors.
+Synthetic monitors go beyond classic end-to-end tests and monitoring practices to ensure that your software continues to work as expected in many conditions. 
+
+Even the most thorough end-to-end tests can't prove your software works because you can't validate all possible states of your application.
+And while improving your monitoring practices could help, it might not be sufficient for features that don't get enough traffic.
+
+Synthetic monitoring solves these problems.
+Start by building _synthetic journeys_, end-to-end tests executed by a real browser that you can run both on your machine and CI.
```

Comment:
I like that we're including this in the docs, but it feels a bit out of place in this intro. Do you think we could have a recommended usage section? Something like "best practices" maybe? Putting this in there could be helpful for readers to compartmentalise the content in their heads.

Reply:
I just removed this. After rewriting the paragraphs before it based on your suggestions above, I didn't think it was necessary.